composer.json

```diff
@@ -5,7 +5,7 @@
     "homepage": "https://airbrake.io",
     "license": "MIT",
     "require": {
-        "php": ">=5.4"
+        "php": "^5.4.7 || ^7.0"
```

Comment:
Yeah, I think we can drop PHP 5.4 since I plan to use guzzle and it requires PHP 5.5+ https://github.com/guzzle/guzzle/blob/master/composer.json#L16